app/routes/errors.py

```diff
@@ -41,11 +46,16 @@ def _render_error_page(status_code, template=None):
     )
 
 
-@errors_blueprint.app_errorhandler(401)
-@errors_blueprint.app_errorhandler(CSRFError)
 @errors_blueprint.app_errorhandler(NoTokenException)
+@errors_blueprint.app_errorhandler(CSRFError)
+@errors_blueprint.app_errorhandler(NoQuestionnaireStateException)
+@errors_blueprint.app_errorhandler(401)
```

Comment:
Minor, but I can't see why the ordering changed here.
```suggestion
@errors_blueprint.app_errorhandler(401)
@errors_blueprint.app_errorhandler(CSRFError)
@errors_blueprint.app_errorhandler(NoTokenException)
@errors_blueprint.app_errorhandler(NoQuestionnaireStateException)

```